docker-compose.yml

```diff
@@ -77,6 +77,7 @@ services:
       HAITATON_OAUTH2_CLIENT_ID: https://api.hel.fi/auth/haitatonapidev
       HAITATON_OAUTH2_USER_INFO_URI: https://tunnistamo.test.hel.ninja/openid/userinfo
       #      HAITATON_CORS_ALLOWED_ORIGINS: http://localhost:8000
+      HAITATON_GDPR_DISABLED: ${HAITATON_GDPR_DISABLED:-true}
```

Comment:
Is this '-' on purpose?

Reply:
Yes. https://stackoverflow.com/a/70772707